README.md

```diff
@@ -1,68 +1,256 @@
-EggTimer
-========
+# EggTimer
 
-A GitHub webhook target that merges Pull Requests when they're ready.
+EggTimer is a bot that automatically merges GitHub pull requests when
+they are ready. EggTimer is a node.js script and a GitHub webhook.
 
-![](https://i.stack.imgur.com/RgWvA.png)
+Upon startup and when the configured GitHub repository changes, the bot
+finds pull requests that are ready for merging and, one by one, triggers
+CI tests and eventually merges successfully tested PRs. Eligible pull
+requests are merged in FIFO (i.e., ascending PR number) order. Pull
+request eligibility and merging steps are detailed further below.
 
-Description
-===========
 
-EggTimer runs as a node script, listening to events (webhooks) from your GitHub repositories. It monitors pull requests, pull request reviews, and changes in status. When the pull request is reviews and approved, and 1 or more tests (checks) have been run, and the pull request is mergeable, then boom!, it's merged! Optionally the branch can be deleted at the same time.
+## Terminology
 
-This eliminates one of the headaches of a reviewer: the minutes and hours spent waiting for tests to finish after code has been reviewed, to merge in the finished branch.
+* _Merging_: Merging a pull request involves many steps, including
+  automated CI tests, PR labeling, and the target branch update. Do not
+  confuse this general/imprecise term with the atomic action of merging
+  one git branch into another.
 
-It's my solution to my own [StackOverflow Question](http://stackoverflow.com/questions/40391344/automatically-merge-verified-and-tested-github-pull-requests/40398678#40398678)
+* _Core developer_: From the bot point of view, a core developer is a
+  GitHub user with write access to the configured repository (`config::repo`).
 
-Configuration
-=============
 
-EggTimer is meant to be run as a web server, which is then called
-by [GitHub's webhook framework](https://developer.github.com/webhooks/).
+## Which pull requests are eligible for merging?
 
-Customize config.js
--------------------
+A pull request is ready for merging if all of the following conditions
+are satisfied:
 
-Create a config.js, by cloning config-example.js:
+* The PR is open (TODO: Check that a PR closed right _after_ we got a
+  list of PRs will not be merged!)
+* The PR has GitHub 'mergeable' status. TODO: Document how a GitHub user
+  can tell whether a PR has a GitHub 'mergeable' status.
```

Comment:
I assume a user can understand this if there is no 'This branch has conflicts...' message. I can add a screenshot to illustrate, if needed.